File: fizzbuzz.py
def main():

    x = 1
    input_angka = int(input("Masukkkan batas angka : "))

    while x <= input_angka:
        if x % 3 == 0 and x % 5 == 0:
            print("Fizzbuzz")
        elif x % 3 == 0:
            print("Fizz")
        elif x % 5 == 0:
            print("Buzz")
        else:
            print(x)
        x += 1
    
    pengulangan = input("Lakukan game lagi? ya / tidak : ").lower
    if pengulangan() == "ya":
        main()
    elif pengulangan() == "tidak":
        print("Ok")
    else:
        print("Saya tidak mengerti perintah tersebut, saya akan menganggap anda menjawab tidak")

main()

File: fizzbuzz.lua
function main()
    x = 1
    
    io.write("Input batas angka : ")
    input = tonumber(io.read())
    
    while x <= input do
        if x % 3 == 0 and x % 5 == 0 then
            print("Fizzbuzz")
        elseif x % 3 == 0 then
            print("Fizz")
        elseif x % 5 == 0 then
            print("Buzz")
        else
            print(x)
        end
        x = x + 1
    end
    io.write("Lakukan game lagi? ya / tidak : ")
    pengulangan = string.lower(io.read())
    
    if pengulangan == "ya" then
        main()
    elseif pengulangan == "tidak" then
        print("Ok")
    else
        print("Saya tidak mengerti perintah tersebut, saya akan menganggap anda menjawab tidak")
    end
end
main()